exercises/rational-numbers/.meta/src/reference/java/Rational.java

```diff
@@ -0,0 +1,102 @@
+public class Rational {
```

Comment:
This could be [`package private`](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html) 🙂